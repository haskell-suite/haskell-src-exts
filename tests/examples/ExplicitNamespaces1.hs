{-# LANGUAGE ExplicitNamespaces #-}
import Data.Array.Repa ( type (++) )
import Data.Array.Repa ( type {- comment here -} (++) )
import Data.Array.Repa (    type {- comment2 here -} (++)  {- and here -} )
