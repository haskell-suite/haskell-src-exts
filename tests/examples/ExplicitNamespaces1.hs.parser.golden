ParseOk (Module (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 1 1 5 1, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 1 1 1 1,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 1 2 1,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 1 2 1,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 1 2 1,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 1 3 1,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 1 4 1,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 5 1 5 1,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 5 1 5 1]}) Nothing [LanguagePragma (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 1 1 1 36, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 1 1 1 13,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 1 33 1 36]}) [Ident (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 1 14 1 32, srcInfoPoints = []}) "ExplicitNamespaces"]] [ImportDecl {importAnn = SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 1 2 37, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 1 2 7]}, importModule = ModuleName (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 8 2 23, srcInfoPoints = []}) "Data.Array.Repa", importQualified = False, importSrc = False, importPkg = Nothing, importAs = Nothing, importSpecs = Just (ImportSpecList (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 24 2 37, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 24 2 25,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 36 2 37]}) False [IVar (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 26 2 35, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 26 2 30,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 31 2 35]}) True (Symbol (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 31 2 35, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 31 2 32,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 32 2 34,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 2 34 2 35]}) "++")])},ImportDecl {importAnn = SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 1 3 56, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 1 3 7]}, importModule = ModuleName (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 8 3 23, srcInfoPoints = []}) "Data.Array.Repa", importQualified = False, importSrc = False, importPkg = Nothing, importAs = Nothing, importSpecs = Just (ImportSpecList (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 24 3 56, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 24 3 25,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 55 3 56]}) False [IVar (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 26 3 54, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 26 3 30,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 50 3 54]}) True (Symbol (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 50 3 54, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 50 3 51,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 51 3 53,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 53 3 54]}) "++")])},ImportDecl {importAnn = SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 1 4 76, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 1 4 7]}, importModule = ModuleName (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 8 4 23, srcInfoPoints = []}) "Data.Array.Repa", importQualified = False, importSrc = False, importPkg = Nothing, importAs = Nothing, importSpecs = Just (ImportSpecList (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 24 4 76, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 24 4 25,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 75 4 76]}) False [IVar (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 29 4 58, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 29 4 33,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 54 4 58]}) True (Symbol (SrcSpanInfo {srcInfoSpan = SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 54 4 58, srcInfoPoints = [SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 54 4 55,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 55 4 57,SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 57 4 58]}) "++")])}] [],[Comment True (SrcSpan "tests/examples/ExplicitNamespaces1.hs" 3 31 3 49) " comment here ",Comment True (SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 34 4 53) " comment2 here ",Comment True (SrcSpan "tests/examples/ExplicitNamespaces1.hs" 4 60 4 74) " and here "])
